config/default.js

```diff
@@ -22,15 +22,23 @@ module.exports = {
                 app: {Console: { timestamp: true, colorize: true, level: 'error' }},
                 http: {},
             },
-            maxAge: 600,
+            cookies: {
+                secure: false,
+                secureProxy: false,
+                httpOnly: false,
+                signed: false,
+                overwrite: true,
+            },
             port: apiPort,
             security: {
+                expirationTokenDelay: 1800, // in seconds
                 bcrypt: {
                     salt_work_factor: 10, // higher is safer, but slower
```

Comment:
Shouldn't we set a higher level for the default config and still use 10 for tests ?